test/mrb_exec.rb

```diff
@@ -28,3 +28,14 @@
   assert_true(p.is_a?(Integer))
   assert_true(status.success?)
 end
+
+assert("Kernel#exec sys fail") do
+  expected_error_message = "sys failed. errno: 2 message: No such file or directory mrbgem message: execv failed"
+
+  begin
+    # always failed
+    p = Process.fork { exec "/bin/bbaash" }
```

Comment:
Will this pass?

In this patch if L39 is ignored, the test will be passed anyway.

```ruby
e = nil
begin
  _p = Process.fork { exec "/bin/bbaash" }
rescue => _e
  e = _e
end
assert_false e.nil?
assert_equal(expected_error_message, e.to_s)
```